questionnaire/non-complex-sexual-assault-id-mapper.json

```diff
@@ -116,5 +116,7 @@
     "applicant-expenses": "p-applicant-expenses",
     "context-money": "p--context-money",
     "applicant-pregnancy-loss": "p-applicant-pregnancy-loss",
+    "dentist-visited": "p-dentist-visited",
+    "dentist-details": "p-dentist-details",
```

Comment:
Could you revise the name of these pages? 'Details' isn't very specific, especially if they decide to elaborate on these questions in the future. Also, should the "entity" in p-dentist-visited be the dentist? It's not really info about a dentist, it's about the applicant visiting a dentist.